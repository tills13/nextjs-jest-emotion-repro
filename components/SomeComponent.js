import styled from "@emotion/styled";

import { Wrapper } from "./WrapperComponent";

function SomeComponent({ className }) {
  return <div className={className}>Some Component</div>;
}

export default styled(SomeComponent)`
  ${Wrapper} & {
    margin-bottom: 10px;
  }
`;
