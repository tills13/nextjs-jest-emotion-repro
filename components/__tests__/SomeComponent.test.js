import SomeComponent from "../SomeComponent";
import { shallow } from "enzyme";

describe("SomeComponent", () => {
  it("renders", () => {
    expect(shallow(<SomeComponent />)).toMatchSnapshot();
  });
});
