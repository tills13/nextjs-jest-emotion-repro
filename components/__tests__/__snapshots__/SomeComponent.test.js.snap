// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent renders 1`] = `
.euyvcx00 .emotion-0 {
  margin-bottom: 10px;
}

<SomeComponent
  className="emotion-0 emotion-1"
/>
`;
